### PR TITLE
Modules create - fuzzy finder for process label

### PR DESCRIPTION
**Pull request by ewels**
New feature suggested by @JoseEspinosa in Slack: https://nfcore.slack.com/archives/CE5LG7WMB/p1615976112010300

Mar-17-2021 11-28-02

### CODE REVIEW

**Review by KevinMenden — APPROVED** [review]
👌

File: nf_core/modules/create.py [hunk 1]
```diff
@@ -210,7 +210,9 @@ def create(self):
             except FileNotFoundError as e:
                 raise UserWarning(f"Could not open 'tests/config/pytest_software.yml' file!")
 
-        log.info("Created module files:\n  " + "\n  ".join(self.file_paths.values()))
+        new_files = list(self.file_paths.values())
+        new_files.append(os.path.join(self.directory, "tests", "config", "pytest_software.yml"))
```
Comment (JoseEspinosa) [inline nf_core/modules/create.py:214]: Is really modified not newly created, though

**Comment by JoseEspinosa** [comment]
Is my OCD sorry :smile:

**Comment by ewels** [comment]
haha, I noticed it as well but was too lazy to do anything about it 😆

**Review by JoseEspinosa — APPROVED** [review]

**Pull request merged**